crates/oxc_transformer/src/es2019/optional_catch_binding.rs

```diff
@@ -1,3 +1,37 @@
+//! ES2019: Optional Catch Binding
+//!
+//! This plugin transform catch clause without parameter to add a parameter called `unused` in catch clause.
+//!
+//! > This plugin is included in `preset-env`, in ES2019
+//!
+//! ## Example
+//!
+//! Input:
+//! ```js
+//! try {
+//!   throw 0;
+//! } catch () {
```

Comment:
I don't think `catch ()` is legal syntax.

```suggestion
//! } catch {
```